content/using_ec2_spot_instances_with_eks/010_prerequisites/prerequisites.files/eks-blueprints.tf

```diff
@@ -159,7 +159,7 @@ module "eks_blueprints_kubernetes_addons" {
 #---------------------------------------------------------------
 module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
-  version = "~> 3.0"
+  version = "~> 4.0"
```

Comment:
Thanks @mseverin for your contribution! Would you mind changing this to a version like `4.0.2` instead? Also, please update your branch from the main one as we recently pushed a new change there. Thanks.

Reply:
Done